New permissions to access smart rules [PAB-4929] (#3854)
SoftwareAG/c8y-docs: content/change-logs/analytics/apama-in-c8y-20251010-Role-based-access-to-Smart-rules.md

---

> date: 2025-10-31

Reviewer: Don't forget to remove the date before you merge this

Author: Will remove it before the final approval.

> title: Smart rules (NEW) introduces support for fine grained RBAC

Reviewer 2: ```suggestion
title: Smart rules (NEW) introduces support for fine-grained RBAC
```

change_type:
  - value: change-pXAlHAWka

>     label: Preview

Reviewer: Ideally, I would call this a `Feature` but since the current state of the plugin itself is in public preview, we can leave this in preview phase too

Author: Ok leaving this as it is.

product_area: Analytics
component:
  - value: component-M5-cepIIS
    label: Streaming Analytics
build_artifact:
  - value: tc-KXXmo2SUR
    label: apama-in-c8y
ticket: PAB-4929

> version: 

Reviewer: 26.247.0

---

> Smart rules (NEW) introduces support for fine grained RBAC.

Reviewer 2: ```suggestion
The smart rules (NEW) feature has introduced support for fine-grained RBAC.
```

> The Smart rules (NEW) now supports fine-grained Role-Based Access Control (RBAC) to manage user permissions more precisely.

Reviewer: plugin?

Author: Should I mention it explicitly as for plugin.

Author: It would be too specific as we can also use these to update the templates from "Streaming analytics" as well, Though both are same. 
It would be better if we keep it as "Smart rules (NEW)".

> The Smart rules (NEW) now supports fine-grained Role-Based Access Control (RBAC) to manage user permissions more precisely.

Reviewer 2: ```suggestion
The smart rules (NEW) feature now supports fine-grained Role-Based Access Control (RBAC) to manage user permissions more precisely.
```

> For more details refer to [Role-based Access ](/streaming-analytics/smart-rules-plugin/#role-based-access).

Reviewer 2: ```suggestion
For details, refer to [Role-based access](/streaming-analytics/smart-rules-plugin/#role-based-access).
```

> For details, refer to [Managing permissions to Smart rules (NEW) instances](/streaming-analytics/smart-rules-plugin/#managing-permissions-to-smart-rules-instances).

Reviewer 2: ```suggestion
For details, refer to [Managing permissions to smart rules (NEW) instances](/streaming-analytics/smart-rules-plugin/#managing-permissions-to-smart-rules-instances).
```
